src/components/clusters/wizards/osd/Networking/ClusterProxy.test.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Formik } from 'formik';
+
+import {
+  HTTP_PROXY_PLACEHOLDER,
+  HTTPS_PROXY_PLACEHOLDER,
+} from '~/components/clusters/common/networkingConstants';
+import { FieldId } from '~/components/clusters/wizards/osd/constants';
+import { render, screen } from '~/testUtils';
+
+import { ClusterProxy } from './ClusterProxy';
+
+jest.mock('@patternfly/react-core', () => ({
+  ...jest.requireActual('@patternfly/react-core'),
+  useWizardContext: jest.fn(() => ({
+    goToStepByName: jest.fn(),
+    activeStep: { name: 'mockStepId' },
+  })),
+}));
+
+const buildTestComponent = (formValues = {}) => (
```

Comment:
total nit:  i think we usually write these methods as 'renderSomething`.

that is, instead of calling `render(buildTestComponent())` everywhere (and since it's the only way it's being used), just change the function impl' to include `render()` and rename it to `renderTestComponent`.